test/test_argparsewrapper.py

```diff
@@ -0,0 +1,41 @@
+import io
+import unittest
+from contextlib import redirect_stderr
+
+from conjur.argparsewrapper import ArgparseWrapper
+
+class ArgparserWrapperTest(unittest.TestCase):
+    capture_stream = io.StringIO()
+    arg_parse = ArgparseWrapper(description='Conjur CLI', add_help=False)
+
+    subparser = arg_parse.add_subparsers(dest='resource', title="Commands")
+    variable_parser = subparser.add_parser('testCommand',
+        help='Manage testCommand')
+    variable_subparsers = variable_parser.add_subparsers(dest='action')
+    variable_subparsers.add_parser('testSubcommand')
+
+    def test_error_prints_help_and_exits(self):
+        with self.assertRaises(SystemExit):
+            with redirect_stderr(self.capture_stream):
+                self.arg_parse.error("unrecognized arguments")
+
+        self.assertEquals(self.capture_stream.getvalue(), "Error unrecognized arguments\n")
```

Comment:
Remove the \n from the right side and use strip function. It will be more readable
```suggestion
        self.assertEquals(self.capture_stream.getvalue().strip(), "Error unrecognized arguments")
```

Reply:
removed the \n all together